### ISSUE TITLE
Why the n_classes is 2?

### ISSUE THREAD

**Issue by artaasd95** [ShawnBIT/UNet-family#4]
Hi, 
I tried to train the model using a pixel wise criterion(L1 loss), my input image has 3 channels and since the final conv2d layer is using n_classes(by default is set to 2), for out_channels, the output has 2 channels and the criterion can not compare these images. Why should out_channels should be 2? Can it change to 3?
the code:
`self.final = nn.Conv2d(filters[0], n_classes, 1)`

**Comment by zacario-li** [comment]
@artadp you should not compare the input image with the network's output. compare the network's output with your label.
